packages/core/contracts/oracle/implementation/Voting.sol

```diff
@@ -22,7 +24,14 @@ import "@openzeppelin/contracts/cryptography/ECDSA.sol";
  * @title Voting system for Oracle.
  * @dev Handles receiving and resolving price requests via a commit-reveal voting scheme.
  */
-contract Voting is Testable, Ownable, OracleInterface, VotingInterface {
+contract Voting is
+    Testable,
+    Ownable,
+    OracleInterface,
+    OracleAncillaryInterface,
+    VotingInterface,
+    VotingAncillaryInterface
+{
```

Comment:
the `Voting` contract now considers the two new interfaces that define the Ancillary voting data.